packages/logger/package.json

```
@@ -0,0 +1,65 @@
+{
+	"name": "@sapphire/plugin-logger",
+	"version": "0.0.1",
+	"description": "Plugin for @sapphire/framework to have pretty console output",
+	"author": "@sapphire",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"exports": {
+		".": {
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js"
+		},
+		"./register": {
+			"import": "./dist/register.mjs",
+			"require": "./dist/register.js"
+		}
+	},
+	"homepage": "https://github.com/sapphire-project/plugins/tree/main/packages/logger",
+	"scripts": {
+		"test": "jest",
+		"lint": "eslint src tests --ext ts --fix",
+		"build": "tsc -b src",
+		"postbuild": "run-p esm:**",
+		"esm:register": "gen-esm-wrapper dist/register.js dist/register.mjs",
+		"esm:default": "gen-esm-wrapper dist/index.js dist/index.mjs"
+	},
+	"dependencies": {
+		"@sapphire/time-utilities": "^1.2.1",
+		"colorette": "^1.2.1"
+	},
+	"peerDependencies": {
+		"@sapphire/framework": "1.x"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/sapphire-project/plugins.git",
+		"directory": "packages/logger"
+	},
+	"files": [
+		"dist",
+		"!dist/tsconfig.tsbuildinfo"
+	],
+	"engines": {
+		"node": ">=12",
```

```suggestion
		"node": ">=14",
```

Can you also check API?